### DISCUSSION TITLE
How to add clickable item to TabBar or FlyoutItem?

### DISCUSSION THREAD

**Discussion by SergTomcat** [dotnet/maui#11149] [Q&A]
I need to add item to TabBar (or FlyoutItem) that is similar to MenuItem in shell (just button with no ContentPage).

```
<FlyoutItem>
		<ShellContent Route="kitties" Title="Kitties" Icon="cat.png" ContentTemplate="{DataTemplate views:GeneralAnimalPage}"/>

		<!-- something like this -->
		<SomeItem Title="Press me"  Click="Click_Handler"/>
		
		<!-- or this -->
		<Tab Title="Press me"  Command="{Binding MyCommand}"/>

		<!-- or even this: no contenttemplate, just routing! -->
		<ShellContent Title="Press me"  Route="myroutepage1"/>


</FlyoutItem>
```
Getting ahead: I will need to add it programmatically in codebehind.

**Comment by PrakashVjr** [comment]
You can try this in your appshell.xaml 
` <MenuItem Text="Home" IconImageSource="home_24dp.png" Command="{Binding Homecommand}"/>`
and write the button-click on your appShell.cs Like the Regular page code behind.

I tried it it Worked for me But the issue in this tab bar and menu item are independent Of each other 

The Tab Bar Item uses the routing Method But the menu item uses the navigation Like GoToAsync or may be navigation async
